npm/angular/src/mount.ts

```diff
@@ -278,21 +278,22 @@ function setupComponent<T extends { ngOnChanges? (changes: SimpleChanges): void
  * @param {Type<T> | string} component Angular component being mounted or its template
```

Comment:
These types in the '@param' can be inferred from the TS code and could be removed to avoid duplication.